lib/cache/fetchAndExtract.js

```diff
@@ -38,7 +38,7 @@ module.exports = async ({ url, name, tmpLoc, outLoc, raw, requestOpts }) => {
                 )
                 .on('end', async () => {
                     try {
-                        await rename(tmpLoc, outLoc)
+                        await fs.move(tmpLoc, outLoc, { overwrite: true })
```

Comment:
I think this `{ overwrite: true }` should be unnecessary now?

Reply:
Verified, it was useless now.

4615d38